### DISCUSSION TITLE
periodic boundary conditions in 1 d and hermite elements

### DISCUSSION THREAD

**Discussion by aquillen** [kinnala/scikit-fem#1153] [Q&A] [answered]
I tried to see if I could make a periodic boundary in 1d (so far so good) until I tried to set up a biharmonic equation.

I tried the following:

<img width="1109" alt="Screenshot 2024-07-31 at 9 21 36 PM" src="https://github.com/user-attachments/assets/49b59521-5adc-41e5-80e2-5879c51b835d">
<img width="1104" alt="Screenshot 2024-07-31 at 9 22 31 PM" src="https://github.com/user-attachments/assets/27a7ceaf-686a-4241-b17f-d0bd59e58506">

Is there a way to do this properly or is the hermite element not supported for the DG meshes?

**Comment by kinnala** [comment, marked as answer]
Not compatible, here is an existing issue: https://github.com/kinnala/scikit-fem/issues/902

You may try other approaches for periodicity such as skfem.utils.mpc or implementing some penalty-type methods.

**Reply by aquillen** [reply]
good to know!  thanks!
